### DISCUSSION TITLE
Welcome to gctoolkit Discussions!

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/gctoolkit#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kittylyst** [comment]
Congrats, folks. This is really great to see, especially after all these years! 😸

**Reply by ghost** [reply]
Thank you Ben...

**Comment by anbusampath** [comment]
Congratulations Team! Looking forward to learn about the tool.

**Comment by Anshul-Gairola** [comment]
Great work team! I am very excited and looking forward to utilise this library.
